tests/unit/ui/components/TestUserReview.js

```diff
@@ -30,6 +33,23 @@ describe(__filename, () => {
     return shallowUntilTarget(<UserReview {...props} />, UserReviewBase);
   };
 
+  const renderAndMount = (otherProps = {}) => {
```

Comment:
We shouldn't have to use `mount` to write these tests. Because we're now embedding a `ShowMoreCard`, instead of asserting about `find('.UserReview-body').html()` we'll need to write assertions about the `children` of the `ShowMoreCard`.